tensorflow_model_optimization/g3doc/guide/clustering/clustering_example.ipynb

```diff
@@ -0,0 +1,647 @@
+{
```

Comment:
<p>nit: targetted -&gt; targeted</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Thanks. Targetted is the British spelling. Now I wonder whether my spell check was in British or American English. I'd better keep it consistent.</p>

Reply:
Oh I'm sorry, I did not know that, thought it was a typo 😅

Reply:
Let's be consistent with tf.org when it's ambiguous. 

https://www.tensorflow.org/s/results?q=targetted uses targeted

Reply:
@TamasArm No need to be sorry. That was well spotted!
@alanchiao Absolutely! Targeted it is, then.